content/en/docs/tasks/manage-kubernetes-objects/kustomization.md

```diff
@@ -834,9 +834,9 @@ spec:
 Kustomize has the concepts of **bases** and **overlays**. A **base** is a directory with a `kustomization.yaml`, which contains a
 set of resources and associated customization. A base could be either a local directory or a directory from a remote repo,
 as long as a `kustomization.yaml` is present inside. An **overlay** is a directory with a `kustomization.yaml` that refers to other
-kustomization directories as its `bases`. A **base** has no knowledge of an overlay and can be used in multiple overlays.
-An overlay may have multiple bases and it composes all resources
-from bases and may also have customization on top of them.
+kustomization directories as its `resources`. A **base** has no knowledge of an overlay and can be used in multiple overlays.
```

Comment:
Shouldn't we remove the subsequent line about bases as well if we're talking about resources and continue with the explanation about resources? Or we should retain the current sentence as-is & mention that from v2.1.0 onwards, we should be using resources instead.

Reply:
Thanks, I deleted the subsequent line